src/controllers/footage.controller.ts

```diff
@@ -153,7 +133,8 @@ export const getFootage = defaultEndpointsFactory.build({
 export const getUserFootage = defaultEndpointsFactory.build({
   method: 'get',
   input: z.object({
-    discordId: z.number(),
+    // had to change to string because the param is sent as a string not as a number for some reason.
+    discordId: z.string(),
```

Comment:
so mongo expects a number, idk why, i think we can use a transform to make it a number https://github.com/RobinTail/express-zod-api#transformations